introduction/de/05-principles.asciidoc

```diff
@@ -0,0 +1,70 @@
+Die Kontaktinformationen des Hostteams sollten in allen Kanälen zur Verfügung stehen die für das Projekt sinnvoll sind.
+Die Absichten des Hostteams InnerSourcebeiträge zu seinem Projekt anzunehmen, sollten über relevante Organisationskanäle mitgeteilt werden, um das Bewusstsein darum to erhöhen.
+Insbesondere in kleineren Organizations möchten Sie möglicherweise regelmäßig über die InnerSourcearbeit Ihres Teams berichten.
+In größeren Unternehmen/Organizationen können solche Berichte/Nachrichten jedoch überwaltigend sein und es ist möglicherweise angemessener sicherzustellen, dass das Projekt z.B. in einem benutzerfreundlichen Tool leicht auffindbar ist.
+Denken Sie daran, das Ziel ist es, die Kanäle zu nutzen, die in IHREM Unternehmen am besten funktionieren.
+
+Die obige Liste ist keineswegs erschöpfend.
+Die Offenheit eines Projekts hängt in der Regel direkt davon ab wie erfolgreich ein Projekt in Bezug auf InnerSource ist.
+Je offener es ist, desto weniger Hindernisse werden für potenzielle Mitwirkende geschaffen.
+Je weniger offen es ist, desto schwieriger wird es für jemanden ausserhalb des Hostteams einen Beitrag zu leisten.
+
+=== Transparenz
+
+Damit Gastteams einen sinnvollen Beitrag zu einem Projekt leisten können, muss sich das Hostteam transparent verhalten.
+Dies bedeutet, dass Gastteams in der Lage sein sollten, folgendes zu verstehen:
+
+* Das Projekt / Repo und seine Ziele
+* Noch offene Funktionsanforderungen
+* Fortschritte bei den Funktionentwicklungen
+* Entscheidungsfindungsprozess des Hostteams
+
+Wenn möglich, sollte das obige klar und detailliert kommuniziert werden, von den internen Definitionen der verbleibenden Arbeit bis hin zu projektspezifischen Spezialszenarien.
+Diese Kommunikation sollte auf eine Weise erfolgen sodass die Information für diejenigen, die nicht Teil des Kernteams sind, leicht gefunden and nachvollzogen werden kann.
+
+=== Priorisierte Betreuung
+
+_Mentorship_ vom Hostteam zum Gastteam durch https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[Trusted Committers] ist ein Schlüsselaspekt von InnerSource.
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[Contributors] in Gastteams sind auf dem neuesten Stand, sodass sie genug über das Projekt / Repo des Host-Teams wissen um es erfolgreich verändern zu können.
+Dabei lernen sie die Software des Hostteams sowohl als genereller Nutzer als auch als Botschafter für das Projekt / die Software besser verstehen.
+Diese einzelnen Mitarbeiter können im Laufe der Zeit und mit wachsender Erfahrung eine breitere Rolle im Projekt als Trusted Committer zu übernehmen.
+
+Es ist wichtig, dass die Schulung für potentielle Contributors vom Hostteam priorisiert wird.
```

Comment:
```suggestion
Es ist wichtig, dass Mentoring für potentielle Contributoren vom Hostteam priorisiert wird.
```

Reply:
Schulung has a special meaning at least in my German book. The picture that comes to my mind is a group of people being educated by one or two teachers.